amoro-mixed-format/amoro-mixed-format-spark/amoro-mixed-format-spark-3-common/src/test/java/org/apache/amoro/spark/test/MixedTableTestBase.java

```diff
@@ -100,7 +100,7 @@ public void cleanUpSource() {
       return;
     }
     if (TestIdentifier.SOURCE_TYPE_ARCTIC.equalsIgnoreCase(source.sourceType)) {
```

Comment:
SOURCE_TYPE_MIXED?

Reply:
The old `arctic` source is kept to be compatible with previous versions.
So are you suggesting changing the variable name to  SOURCE_TYPE_MIXED and keeping the variable value to arctic?